main.py

```diff
@@ -6,32 +6,20 @@
 from depthai_nodes.node.parsing_neural_network import ParsingNeuralNetwork
```

Comment:
I think you can import the node like this:
```suggestion
from depthai_nodes.node import ParsingNeuralNetwork
```